presto-docs/src/main/sphinx/installation/cli.rst

```diff
@@ -14,7 +14,90 @@ make it executable with ``chmod +x``, then run it:
 
     ./presto --server localhost:8080 --catalog hive --schema default
 
-Run the CLI with the ``--help`` option to see the available options.
+The available options for the CLI are:
```

Comment:
I agree, but we can do better than just copy paste, given that this is HTML not just plain text. For example: https://git-scm.com/docs/git-remote